src/client/controllers/index.js

```diff
@@ -51,11 +51,28 @@ const pageMap = {
 };
 
 const Child = pageMap[page] || Index;
+let extraIndexProp;
+
+if (Child === Index) {
+	if (props.user !== null) {
+		extraIndexProp = {
+			isLoggedIn: true
+		};
+	}
+	else {
+		extraIndexProp = {
+			isLoggedIn: false
+		};
+	}
+}
+else {
+	extraIndexProp = ' ';
+}
 
 const markup = (
 	<AppContainer>
 		<Layout {...extractLayoutProps(props)}>
-			<Child {...extractChildProps(props)}/>
+			<Child {...extractChildProps(props)} {...extraIndexProp}/>
```

Comment:
@MonkeyDo i got your point!
But i would like to make another pr for this, beacuse of clear intention to show the issue we found as a ticket..

Reply:
@MonkeyDo one more thing 
I don't think there is any need of disableSignup in here any more , it actually used to change the color of signup button which is now in case in removed by my proposal so it must be removed !

Reply:
however it is being used in Layout.js so it must be not removed from everywhere , it must be removed from index.js